src/backends/beamcoder.ts

```diff
@@ -380,6 +381,7 @@ export class BeamcoderExtractor extends BaseExtractor implements Extractor {
                 VERBOSE && console.log('closestFramePTS', closestFramePTS, 'targetPTS', targetPTS);
                 if (!outputFrame || closestFramePTS <= targetPTS) {
                     VERBOSE && console.log('assigning outputFrame', closestFrame?.pts);
+                    this.#previousTargetPTS = targetPTS;
```

Comment:
We must update `previousTargetPTS` each time we update or return `outputFrame`. Otherwise, the system can't know we're looking backwards and thus doesn't seek!

Reply:
Would be good to add this comment here and in the other place